src/modules/missionHelper/i18n/de_DE.root.json

```diff
@@ -116,9 +112,17 @@
                 "description": "Zeigt ob ein KTW anstatt eines RTWs genutzt werden kann.",
                 "title": "KTW statt RTW"
             },
+            "allow_police_motorcycle_instead_of_fustw": {
```

Comment:
We don't need the translations doubled. If needed without the module active => de_DE.root.json. If only needed with module active => de_DE.json

Reply:
please explain this bug

Reply:
I had a quick talk with jan. We search in all files an the xx_XX.json will overwrite the xx_XX.root.json file if both files are present. But since I forgot to compare the key-pair values it's correct